starter_code/queries.md

```diff
@@ -4,80 +4,80 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+<!-- FILTER :{ name : 'Babelgum' }, PROJECT:{ name : 1 } -->
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+<!-- FILTER:{ name : 'Babelgum' }, SORT:{number_of_employees: 1},MAXITEMS:5000,LIMIT:20 -->
```

Comment:
no se muy bien que habéis intentado aquí, no te pedían que el nombre fuera ese. La query sería:
query = {number_of_employees : {$gt : 5000}} sort = {number_of_employees : 1} limit = 20